apisix/plugins/request-id.lua

```diff
@@ -102,13 +106,13 @@ local function gen_data_machine(max_number)
 
                 local _, err3 =
                     etcd_cli:set(
-                    prefix .. tostring(id),
-                    uuid,
-                    {
-                        prev_kv = true,
-                        lease = res.body.ID
-                    }
-                )
+                        prefix .. tostring(id),
+                        uuid,
+                        {
+                            prev_kv = true,
+                            lease = res.body.ID
+                        }
+                    )
 
```

Comment:
Why add this space?

Reply:
@Sn0rt It is defined in the code style document
[CODE_STYLE.md](https://github.com/apache/apisix/blob/master/CODE_STYLE.md)